recipe/meta.yaml

```diff
@@ -1,39 +1,35 @@
 {% set version = "3.2.3" %}
+{% set name = "scs" %}
 
 package:
-  name: scs
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://github.com/bodono/scs-python/archive/refs/tags/{{ version }}.tar.gz
-    sha256: 7592805810fb6da09955dc45322b2d376bca993928c972f472f1a5352821ddcd
-  # include submodule (not in github tarball due to dear-github/dear-github#214)
-  - git_url: https://github.com/cvxgrp/scs.git
-    git_rev: ac6840a3b3264950e6c300264cbf3937e0bcc6c5
-    folder: scs
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e3bd779e7e977e3ae5a2f2035aa4c2a309e29082d59a722d5d6592edc4bdb4b3
 
 build:
+  # Windows is missing libopenblas
+  skip: true  # [py<37 or win]
   number: 0
   script:
-    - export BLAS_LAPACK_LIB_PATHS="$PREFIX/lib"    # [unix]
-    - export BLAS_LAPACK_LIBS="lapack:cblas:blas"   # [unix]
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
-    - libblas
-    - liblapack
+    - libopenblas 0.3.21
     - python
     - setuptools
+    - wheel
     - pip
     - numpy
   run:
     - python
+    - mkl  # [linux64 or (osx and x86)]
+    - libopenblas
```

Comment:
I think it should be liboopenblas for unix only. And mkl for win.

Reply:
`mkl` is pulled in by `numpy` for x86 platforms and in turn used by `scs`, so leaving it out causes an overlinking error.